From d9703831affac962309ab175e892d4d1ab1c74a9 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 21 Apr 2021 16:15:56 +0300
Subject: [PATCH] [SW-28697] module_lkd: use enum for ram_type

The module_lkd_send_status() function for copy/paste reasons was using
the unsigned int for the ram_type parameter, while there is an
established enum for it.
Use the enum for ram_type.

Change-Id: I35456339040adcbc6f972e3c2666d2a36ebf0ea8
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_lkd.c b/preboot/module_lkd.c
index 614a084..e6255bd 100644
--- a/preboot/module_lkd.c
+++ b/preboot/module_lkd.c
@@ -47,7 +47,8 @@
 
 __weak void preboot_disable_iatu_access(void) {}
 
-static void module_lkd_send_status(enum comms_sts sts, unsigned int ram_type,
+static void module_lkd_send_status(enum comms_sts sts,
+				   enum comms_ram_types ram_type,
 				   unsigned int offset)
 {
 	struct comms_status status;
